pyalgostrategypool/reverse_rsi_crossover_v2.py

```diff
@@ -0,0 +1,79 @@
+"""
+    checkout:
+        - strategy specific docs here : https://algobulls.github.io/pyalgotrading/strategies/reverse_rsi/
+        - generalised docs in detail here : https://algobulls.github.io/pyalgotrading/strategies/common_regular_strategy/
+"""
+
+
+class ReverseRSICrossover(StrategyBase):
+    name = 'Reverse RSI v2'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.time_period = self.strategy_parameters['TIME_PERIOD']
+        self.overbought_value = self.strategy_parameters['OVERBOUGHT_VALUE']
+        self.oversold_value = self.strategy_parameters['OVERSOLD_VALUE']
+
+        self.main_order_map = None
+
+    def initialize(self):
+        self.main_order_map = {}
+
+    def get_crossover_value(self, instrument):
+        hist_data = self.get_historical_data(instrument)
+
+        rsi_value = talib.RSI(hist_data['close'], timeperiod=self.time_period)
+
+        oversold_list = [self.oversold_value] * rsi_value.size
+        overbought_list = [self.overbought_value] * rsi_value.size
+
+        oversold_crossover_value = self.utils.crossover(rsi_value, oversold_list)
+        overbought_crossover_value = self.utils.crossover(rsi_value, overbought_list)
+
+        return oversold_crossover_value, overbought_crossover_value
+
+    def strategy_select_instruments_for_entry(self, candle, instruments_bucket):
+        selected_instruments, meta = [], []
+
+        for instrument in instruments_bucket:
+            if self.main_order_map.get(instrument) is None:
+                oversold_crossover_value, overbought_crossover_value = self.get_crossover_value(instrument)
+
+                if oversold_crossover_value == 1:
+                    selected_instruments.append(instrument)
+                    meta.append({'action': 'BUY'})
+
+                elif overbought_crossover_value == -1:
+                    selected_instruments.append(instrument)
+                    meta.append({'action': 'SELL'})
+
+        return selected_instruments, meta
+
+    def strategy_enter_position(self, candle, instrument, meta):
+        self.main_order_map[instrument] = _ = self.broker.OrderRegular(instrument, meta['action'], quantity=self.number_of_lots * instrument.lot_size)
+        return _
+
+    def strategy_select_instruments_for_exit(self, candle, instruments_bucket):
+        instruments, meta = [], []
```

Comment:
here, it is 
`instruments, meta = [], []`

check above comment

this should be consistent. make it 

`instruments, meta = [], []`

everywhere

Reply:
I have kept selected instruments every where?

Reply:
done